df.history.xml

```diff
@@ -2507,20 +2507,24 @@
         <stl-vector name='unk1' type-name='int32_t'/>
         <stl-vector name='unk2' type-name='int16_t'/>
 
-        <int32_t/>
+        <int32_t name='total_unk'/> some value during worldgen, and at the end equals to the number of entities plus that value
         <int32_t name='total_powers' comment='also includes megabeasts'/>
         <int32_t name='total_megabeasts'/>
         <int32_t name='total_semimegabeasts'/>
         <stl-vector/>
         <static-array name='unk_v42_1' type-name='int16_t' count='28' since='v0.42.01'/> unsure but 16-bit values make more sense than 32-bit
-        <stl-vector name='unk_histfig_1' pointer-type='historical_figure'/>
-        <stl-vector name='unk_histfig_2' pointer-type='historical_figure'/>
-        <stl-vector name='unk_histfig_3' pointer-type='historical_figure'/>
+
+        <stl-vector name='live_megabeasts' pointer-type='historical_figure'/>
+        <stl-vector name='live_semimegabeasts' pointer-type='historical_figure'/>
+        <stl-vector name='unk_histfig_3' pointer-type='historical_figure'/> 
         <stl-vector name='unk_histfig_4' pointer-type='historical_figure'/>
         <stl-vector name='unk_histfig_5' pointer-type='historical_figure'/>
+
+        11 - necromancers
         <static-array name='unk_v40_1' since='v0.40.01' count='13'>
```

Comment:
Maybe the other vectors should be placed in a compound with these, or this array should be unrolled. It would be useful to have the "necromancers" name visible.

Reply:
I assumed just these 13 have been placed in an array for a reason. Ah, yes, also they indeed look special - all unk_histfig_ vectors have values, but I've NEVER seen any of these 13 being non-empty apart from necromancers.